index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <script src="index.js" defer></script>
+    <title>The ethics of AI</title>
+  </head>
+  <body>
+    <section class="DOM-menu--inactive"></section>
+    <!-- Desktop view start -->
+    <section class="header-d">
+      <ul class="header-d__ul">
+        <li>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="18"
+            height="18"
+            viewBox="0 0 18 18"
+            fill="none"
+          >
+            <path
+              d="M5.65426 18C12.4466 18 16.1633 11.0728 16.1633 5.07579C16.1633 4.8809 16.1633 4.68602 16.1561 4.49114C16.8764 3.85335 17.503 3.04724 18 2.13484C17.3373 2.49803 16.6243 2.73721 15.8752 2.85236C16.6387 2.29429 17.2221 1.39961 17.503 0.336614C16.7899 0.859251 15.9976 1.2313 15.1549 1.43504C14.4778 0.549212 13.5198 0 12.461 0C10.4226 0 8.76591 2.0374 8.76591 4.54429C8.76591 4.89862 8.80192 5.24409 8.85954 5.58071C5.79112 5.39468 3.06843 3.57874 1.2461 0.832677C0.929172 1.50591 0.7491 2.28543 0.7491 3.11811C0.7491 4.69488 1.40456 6.08563 2.39136 6.90059C1.78631 6.87402 1.21729 6.67027 0.720288 6.33366C0.720288 6.35138 0.720288 6.36909 0.720288 6.39567C0.720288 8.59252 1.9952 10.435 3.68067 10.8514C3.37095 10.9577 3.04682 11.0108 2.70828 11.0108C2.47059 11.0108 2.2401 10.9843 2.01681 10.9311C2.48499 12.7382 3.85354 14.0492 5.46699 14.0846C4.19928 15.3071 2.60744 16.0335 0.878752 16.0335C0.583434 16.0335 0.288115 16.0157 0 15.9715C1.62785 17.247 3.57263 18 5.65426 18Z"
+              fill="white"
+            />
+          </svg>
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </section>
+    <nav class="navbar">
+      <img class="navbar-logo" src="./img/logo.png" alt="logo" />
+      <ul class="navbar__ul">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="li-special"><a href="#">CC Campaign</a></li>
+      </ul>
+    </nav>
```

Comment:
- [ ] please, like the previous reviewer noted, kindly ensure and check well that the hamburger menu is visible in the mobile version as it helps to move from one page to another as you did on about page. kindly check if you pushed all your code. 
- [ ] also, the previous reviewer noted that you should consider not displaying the logo on mobile version. Remember it is someone's design, and you were to personalize the content but follow the design. Please, following the design guidelines is always important. 
![demo33](https://user-images.githubusercontent.com/65298048/226113915-1982416d-eb65-473e-a185-e5412cad2573.gif)